platform/darwin/src/MGLGeoJSONSource.h

```diff
@@ -7,48 +7,53 @@ NS_ASSUME_NONNULL_BEGIN
 @protocol MGLFeature;
 
 /**
+ Options for `MGLGeoJSONSource` objects.
+ */
+typedef NSString *MGLGeoJSONSourceOption NS_STRING_ENUM;
+
+/**
  An `NSNumber` object containing a Boolean enabling or disabling clustering.
  If the `features` property contains point features, setting this option to
  `YES` clusters the points by radius into groups. The default value is `NO`.
  */
-extern NSString * const MGLGeoJSONClusterOption;
+extern MGLGeoJSONSourceOption const MGLGeoJSONClusterOption;
```

Comment:
`MGLGeoJSONSourceOption` combined with `MGLGeoJSON{Foo}Option` will make it look like `MGLGeoJSONSourceOption.bufferOption`. Can we use the root `MGLGeoJSONSourceOption` plus noun (i.e. Cluster, ClusterRadius, etc.) and "limit our options"? 😄